### DISCUSSION TITLE
How to upgrade packages using trex cli?

### DISCUSSION THREAD

**Discussion by borsemayur2** [crewdevio/Trex#91] [Q&A] [answered]
Can we upgrade all or some of the packages using trex to their latest versions?

```bash
trex upgrade *  // Upgrade all packages to latest versions

trex upgrade oak, denodb  // Upgrade denodb and oak to latest versions
```

**Comment by buttercubz** [comment, marked as answer]
hello, to update the packages there are two ways the first is to install the latest version available using the `install` command.

`example`

```console
trex install -m oak
```
this will install the latest version available in `deno.land`, Personally, I don't recommend using this as you download the package without using a specific version.

url in import_map.json `https://deno.land/x/oak.mod.ts`

> also you can use the short method: `trex i -m oak`

The second way and which is the one I recommend to use, is always to specify which version to update to.

`example`

```console
trex install -m oak@v7.2.0
```
in this way download this specific version, in the same way you can update several packages in the same command.

`example`

```console
trex i -m oak@v7.2.0 denodb@v1.0.30
```

**Reply by borsemayur2** [reply]
Okay. Thanks
